docs/release-notes/1.1.0/release-1.1.0.md

```diff
@@ -0,0 +1,116 @@
+# [ML.NET](http://dot.net/ml) 1.1.0 
+## **New Features**
+- **Image type support in IDataView**  
+   [PR#3263](https://github.com/dotnet/machinelearning/pull/3263) added support
+  for in-memory image as a type in IDataView. Previously to consume image it was
```

Comment:
Perhaps rewrite for clarity, for example: Previously it was not possible to use an image directly in IDataView, and the user had to specify the file path as a string and load the image using a transform. The feature resolved the following issues:

Reply:
Thanks, looks great!